kong/plugins/ldap-auth/access.lua

```diff
@@ -68,8 +68,7 @@ local function authenticate(conf, given_credentials)
   if given_username == nil then
     return false
   end
-
-  local credential = cache.get_or_set(cache.ldap_credential_key(given_username), function()
+  local credential = cache.get_or_set(cache.ldap_credential_key(ngx.ctx.api.name, given_username), function()
```

Comment:
Can you create an integration test at https://github.com/Mashape/kong/blob/master/spec/01-unit/11-database_cache_spec.lua?